azext_iot/central/_help.py

```diff
@@ -319,6 +320,92 @@ def _load_central_api_token_help():
     """
 
 
+def _load_central_cde_help():
+    helps[
+        "iot central app cde"
+    ] = """
+        type: group
+        short-summary: Create and Manage Continuous Data Export (CDE) for an application.
+        long-summary: Continuous Data Export enables exporting IoT Central measurements, devices, and device templates data to storage account.
+    """
+
+    helps[
+        "iot central app cde create"
+    ] = """
+        type: command
+        short-summary: Create a new continuous data export
+        examples:
+        - name: Add new continuous data export
+          text: >
+            az iot central app cde create
+            --app-id {appid}
+            --display-name {displayName}
+            --sources "devices, deviceTemplates, telemetry"
```

Comment:
With the way the sources arg is defined, this example with quotes around it is going to be taken as a single value.